.github/workflows/silabs-firmware-build.yaml

```diff
@@ -94,10 +94,17 @@ jobs:
               patch -p1 < $patch
           done
 
+      - name: Patch Makefile
+        run: |
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
+          meta_datetime=$(git log -1 --format="%cd" --date=format:"%b %-d %Y %H:%M:%S")
```

Comment:
is that even valid? shouldnt it be a locale, like `LC_TIME=en_US.UTF-8`.

Regardless I wouldnt expect it to make a difference given the specific date format specifiers.

Something like this would use the locale, but its a bit of a different format
```
LC_TIME=en_US.UTF-8 meta_datetime=$(git log -1 --format="%cd" --date=format:%c)
```

Reply:
Probably the proper way to do it is using C locale (https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap07.html). So:

```suggestion
          LC_TIME=C meta_datetime=$(git log -1 --format="%cd" --date=format:"%b %-d %Y %H:%M:%S")
```

> Regardless I wouldnt expect it to make a difference given the specific date format specifiers.

It does for me:

```
$ git log -1 --format="%cd" --date=format:"%b %-d %Y %H:%M:%S"
Mär 23 2023 09:48:39
$ LC_TIME=C git log -1 --format="%cd" --date=format:"%b %-d %Y %H:%M:%S"
Mar 23 2023 09:48:39
```